tools/go-cli/build.gradle

```diff
@@ -47,7 +49,7 @@ def run(cmd, ignoreError = false) {
     }
 }
 
-task dump_os() << {
```

Comment:
I made it a function and calling it, I wanted to always see printed and all builds this info, in case we need to troubleshoot a build or user opens issue with output from build.
Before it was deadcode since it was not used or documented in the go-cli/README.md

I can switched it back to a task and have the task run as part of the main build, I can make it a dependency on copyCLIShortcut task. They key thing is I wanted always printed, and this will make it easy to be called like you say.